timemachine/fe/free_energy.py

```diff
@@ -1,22 +1,22 @@
 from dataclasses import dataclass
-from typing import List
+from typing import List, Optional
 
 import numpy as np
 
-from timemachine.fe import model_utils, topology
+from timemachine.fe import model_utils, system, topology
 from timemachine.fe.utils import get_mol_masses, get_romol_conf
 from timemachine.ff import ForcefieldParams
 from timemachine.ff.handlers import openmm_deserializer
-from timemachine.lib import LangevinIntegrator, MonteCarloBarostat
+from timemachine.lib import LangevinIntegrator, MonteCarloBarostat, custom_ops
 from timemachine.lib.potentials import CustomOpWrapper, HarmonicBond
 from timemachine.md.barostat.utils import compute_box_center, get_bond_list, get_group_indices
 
 
+@dataclass
 class HostConfig:
-    def __init__(self, omm_system, conf, box):
-        self.omm_system = omm_system
-        self.conf = conf
-        self.box = box
+    omm_system: system.VacuumSystem  # TODO: rename to TimemachineSystem? (not OpenMM, not vacuum)
```

Comment:
It seems the types are not consistent throughout our code.

For example from builders.build_water_system/builders.build_protein_system I get
omm_system: <class 'simtk.openmm.openmm.System'> 
conf: <class 'simtk.unit.quantity.Quantity'> 
box: <class 'numpy.ndarray'>

Reply:
Good catch! Fixed the `omm_system` annotation in https://github.com/proteneer/timemachine/pull/936/commits/2ff38e6bf317044b415c7940b70a51f0d650b1d5

Reply:
Reverted annotations in https://github.com/proteneer/timemachine/pull/936/commits/ed56d13856e4ac3fab64730c18a80e7396f95bec